docs/README.md

```diff
@@ -1,3 +1,26 @@
+# Schema Registry Documentation
+The full Confluent Platform documentation is available at [docs.confluent.io](https://docs.confluent.io/current/schema-registry/docs/index.html).
+
+
+- [Introduction](intro.rst)
```

Comment:
I probably wouldn't even link to these as they kind of discourage you from going to the real docs. Also, I don't know why GH displays a rendered version of the rst since it doesn't actually render them fully correctly.

Reply:
good point